include/zephyr/dt-bindings/reset/stm32h7_reset.h

```diff
@@ -12,12 +13,18 @@
 /* RCC bus reset register offset */
 #define STM32_RESET_BUS_AHB1  0x80
 #define STM32_RESET_BUS_AHB2  0x84
+#if defined(CONFIG_SOC_SERIES_STM32H7RSX)
+#define STM32_RESET_BUS_AHB3  0xA4
+#define STM32_RESET_BUS_AHB5  0x7C
+#define STM32_RESET_BUS_APB5  0x8C
+#else
 #define STM32_RESET_BUS_AHB3  0x7C
+#define STM32_RESET_BUS_APB3  0x8C
+#endif /* CONFIG_SOC_SERIES_STM32H7RSX */
```

Comment:
KConfig symbols don't work in dts bindings (dts is parsed before Kconfig), a dedicated file is required

Reply:
new _include/zephyr/dt-bindings/reset/stm32h7rs_reset.h_ created instead